Task6.09

```diff
@@ -0,0 +1,19 @@
+import numpy as np
+
+
+def multiply_matrix(first_array, second_array) -> (tuple[int, int], tuple[int, int]):
```

Comment:
Insert argument types.
Please read up on typing, what you have done makes no sense.
https://www.geeksforgeeks.org/duck-typing-in-python/
https://stackoverflow.com/questions/125367/dynamic-type-languages-versus-static-type-languages
https://docs.python.org/3/library/typing.html
https://realpython.com/lessons/pros-and-cons-type-hints/